### PR TITLE
Create invoices endpoints

### PR DESCRIPTION
**Pull request by eduardohertz**

### CODE REVIEW

**Comment by eduardohertz** [comment]
After this PR is merged, a new version can be released

File: lib/ecommerce/resources/invoice_order.rb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+      #
+      # Finds an invoice of an order
+      #
+      # [API]
+      #   Method: <tt>GET /api/orders/:slug/:order_id/invoices/:id/</tt>
+      #
+      #   Documentation: http://myfreecomm.github.io/passaporte-web/ecommerce/api/orders.html#listagem-das-ordens-de-compra-de-uma-conta
+      #
+
```
Comment (rodrigotassinari) [inline lib/ecommerce/resources/invoice_order.rb]: there must be no empty line between a comment and the commented code

**Comment by rodrigotassinari** [comment]
please fix the comments and also hound's  suggestions. the code seems good.

**Comment by eduardohertz** [comment]
all fixed

**Comment by rodrigotassinari** [comment]
:+1:

**Pull request merged**